pkg/build/util/util.go

```diff
@@ -70,3 +71,15 @@ func IsBuildComplete(build *buildapi.Build) bool {
 func BuildNameForConfigVersion(name string, version int) string {
 	return fmt.Sprintf("%s-%d", name, version)
 }
+
+// ConfigSelector returns a label Selector which can be used to find all
+// builds for a BuildConfig.
+func ConfigSelector(name string) labels.Selector {
+	return labels.Set{buildapi.BuildConfigLabel: name}.AsSelector()
```

Comment:
I don't believe we have. You might have seen it in my other PR's that didn't get merged.